lib/puppet/ssl/certificate_factory.rb

```diff
@@ -19,12 +19,11 @@ module Puppet::SSL::CertificateFactory
   # @param serial [Integer] The serial number for the given certificate, which
   #   MUST be unique for the given CA.
   # @param ttl [String] The duration of the validity for the given certificate.
-  #   defaults to Puppet[:ca_ttl]
   #
   # @api public
   #
   # @return [OpenSSL::X509::Certificate]
-  def self.build(cert_type, csr, issuer, serial, ttl = nil)
+  def self.build(cert_type, csr, issuer, serial, ttl = 3600)
```

Comment:
I assume the ttl change here was because we were deprecating `ca_ttl`? Since we're not we can probably undo this change. If we wanted to keep it we'd at least need to change the default (ca_ttl defaults to 5y, not 1hr).

Reply:
Arggh! You changed the file location, and changed content, kept the git history, and the diff is even displayed properly in the web view!!!

You're going to need to show me how you did this, I must be missing some flag in my `git mv`ing repertoire.

Reply:
Uhhh... I did literally nothing. Used shell's built-in `mv` :P